department-of-veterans-affairs/vets-api
# Simple forms api: Remove prefill for form 21-4142 and 21-10210

We decided we wanted to disable prefill in these forms.

[For reference: PR adding prefill for 21-4142](https://github.com/department-of-veterans-affairs/vets-api/pull/12416)
[For reference: PR adding prefill for 21-10210](https://github.com/department-of-veterans-affairs/vets-api/pull/12619)

## Summary

- Remove all prefill related configuration for forms 21-4142 and 21-10210

## Related issue(s)
- https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/346


## Testing done

- Tested locally with vets-website and vets-api running.
  - Logged in as user before changes
  - Verified it was prefilling
  - Removed code
  - Logged in as user after changes
  - Verified it did not prefill

- Tested on deployed branch
  - Signed in
  - Verified no information was prefilled (see screenshot)

## Screenshots
Deployed branch after
<img width="1083" alt="Screenshot 2023-06-21 at 4 13 09 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/57802560/f5b03c15-a396-4cf9-ad53-256a9f3cb902">


## What areas of the site does it impact?
Forms 21-10210 and 21-4142 (not in prod)
